editor/docks/scene_tree_dock.cpp

```diff
@@ -956,6 +958,14 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 
 			undo_redo->commit_action();
 
+			for (KeyValue<Node *, HashMap<Ref<Resource>, Ref<Resource>>> &KV : resources_local_to_scenes) {
+				for (KeyValue<Ref<Resource>, Ref<Resource>> &R : KV.value) {
+					if (R.value->is_local_to_scene()) {
+						R.value->setup_local_to_scene();
+					}
+				}
+			}
```

Comment:
This ensures that local-to-scene resources in newly copied child scene instances are effective when **Editable Children** is enabled.

The cache is now only cleared only when a new copy begins. As users edit the scene, it may lead to the use of released node instances as keys.